command_processor.py

```diff
@@ -298,7 +298,7 @@ def __get_light_status__(self):
                 brightness = "dim."
             elif self.__sensors__.current_light_sensor_reading.lux <= \
                     self.__configuration__.hangar_lit:
-                brightness = "lit."
+                brightness = "lit AF"
```

Comment:
The underlying code needs to know why the hangar is lit AF.

It could be 4 loco, it could be Coors. We just don't know.